### ISSUE TITLE
AssemblyAI - Deep Learning Researcher - Remote

### ISSUE THREAD

**Issue by kbassemblyai** [TechnologyMasters/jobs#507]
### Salary Expectation

- $140K - 200K+ USD (DOE)
- Full Time

### Benefits

- Equity
- 100% Remote team
- Unlimited PTO
- Premium Healthcare (100% Covered)
- Dental and vision care
- $1K budget for your home office setup
- New Macbook Pro (or PC if you prefer)
- 3-4x/year company paid team retreats

### Location

- Remote
- US, Canada, South America, Europe, Australia or remote & willing to work within US time zones

## What You'll Do

Our ASR models outperform big tech companies like Google, AWS, and Microsoft. That being said, there's still a lot of work to do to match human level accuracy. Our Deep Learning team is a tight knit group of creative researchers and engineers, who are not afraid to try unconventional ideas. In this role, you'll:

- Work with large scale datasets to research and train Deep Learning models for Speech Recognition
- Conduct research and experiments in order to improve accuracy of Deep Learning ASR pipelines like CTC, LAS and RNN-Ts
- Dig into weaknesses and failure points of our current ASR models, in order to identify further areas for improvement
- Work with the broader Speech Recognition team to publish papers on novel findings
- Continually push the State of the Art in Speech Recognition to get to human level performance

### Must Have

- 2 - 3 years of experience in Deep Learning research and development
- 2+ years experience with PyTorch or TensorFlow
- Experience training large models on multiple GPUs

### Nice to Have

- Experience with ASR (can be experience from a PhD, Masters, or work experience)
- Experience with fully Deep Learning based ASR systems (CTC, RNNT, Wav2Letter)

### Relevant Experiences

- Startup experience 

## About Our company

AssemblyAI (www.assemblyai.com) is the #1 rated API for Speech Recognition. Thousands of developers use our APIs to transcribe millions of videos, podcasts, phone calls, and zoom meetings every day - powering innovative products like visual voicemail, meeting summarizers, and closed captioning.

We're backed by leading investors including Y Combinator, John and Patrick Collison of Stripe, Daniel Gross, and Nat Friedman. We were founded in 2017.

## How to apply

Please apply the role directly on our careers page: https://apply.workable.com/assemblyai/j/732C9A8A7F/

---

## Meta

- [x] Full Time
- [ ] Part Time
- [x] Contract
- [ ] Internship
- [x] Remote Worldwide
- [x] Remote Regional
- [x] Remote OK
